src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,52 +1,90 @@
 package core.basesyntax;
 
 /**
- * <p>Реалізувати свій ArrayList який імплементує інтерфейс List</p>
+ * Реалізувати свій ArrayList, який імплементує інтерфейс List.
  */
 public class ArrayList<T> implements List<T> {
+    private final int initialCapacity = 10;
```

Comment:
it is not a constant
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names